polar-core/src/vm.rs

```diff
@@ -3123,15 +3120,15 @@ mod tests {
     #[test]
     fn unify_expression() {
         let mut vm = PolarVirtualMachine::default();
-        vm.push_goal(query!(op!(Unify, term!(1), term!(1))))
+        vm.push_goal(query!(op!(Unify, 1.into(), 1.into())))
             .unwrap();
 
         assert_query_events!(vm, [
             QueryEvent::Result{hashmap!{}},
             QueryEvent::Done { result: true }
         ]);
 
-        let q = op!(Unify, term!(1), term!(2));
+        let q = op!(Unify, 1.into(), 2.into());
```

Comment:
imo `term!(..)` here clearly shows what type is being constructed, while `1.into()` is unclear.